src/Covenant/Internal/Ledger.hs

```diff
@@ -653,3 +654,28 @@ a = Abstraction (BoundAt Z ix0)
 
 b :: ValT AbstractTy
 b = Abstraction (BoundAt Z ix1)
+
+-- Datatypes for testing, defined here to avoid having to export the helpers
+
+eitherT :: DataDeclaration AbstractTy
+eitherT =
+  mkDecl $
+    Decl
+      "Either"
+      count2
+      [ Ctor "Left" [Abstraction (BoundAt Z ix0)],
+        Ctor "Right" [Abstraction (BoundAt Z ix1)]
+      ]
+      (PlutusData ConstrData)
+
+unitT :: DataDeclaration AbstractTy
+unitT =
+  mkDecl $
+    Decl
+      "Unit"
+      count0
+      [Ctor "Unit" []]
+      (PlutusData ConstrData)
+
+testDatatypes :: [DataDeclaration AbstractTy]
+testDatatypes = [maybeT, eitherT, unitT, pair]
```

Comment:
I think the better place for these is in the `Covenant.Test` module honestly. They're test helpers, and thus should live with the other test helpers.

Reply:
Ok. I'll have to export the helpers (but tbh maybe that's not such a bad thing).